nipy/algorithms/graph/graph.py

```diff
@@ -680,7 +680,7 @@ def dijkstra(self, seed=0):
         dist, active = np.inf * np.ones(self.V), np.ones(self.V)
         idx, neighb, weight = self.compact_neighb()
         dist[seed] = 0
-        dg = zip(np.zeros_like(seed), seed)
+        dg = list(zip(np.zeros_like(seed), seed))
```

Comment:
zip in PY2 returns a list, so listing a list is a waste.  you could  `from six.moves import zip`